src/components/GuideSelect.tsx

```diff
@@ -8,21 +8,25 @@ export interface GuideSelectProps {
   readonly igs?: string[];
 }
 
-export function GuideSelect({ igs }: GuideSelectProps) {
+export function GuideSelect({ igs }: GuideSelectProps): React.ReactElement {
```

Comment:
I'm partial to importing these types directly.  Looks cleaner IMO and we typically do that for other imports (like useContext in this file)

i.e. 

```typescript
import React, { useContext, ReactElement } from 'react';
...
export function GuideSelect({ igs }: GuideSelectProps): ReactElement {
```